dev/model.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+class Location(models.model):
+    location = models.ForeignKey('self', on_delete=models.SET_NULL)
```

Comment:
Please don't modify database objects without direct interaction. This should also match for on_delete. I prefer not to allow deletion of locations if some other still points on it, instead of relieving it into the void.

Reply:
The Django therm I mean ist `models.PROTECT`